Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -233,6 +233,14 @@ public class SchemeGenerator {
             }
             locationScenarioReference = XCScheme.LocationScenarioReference(identifier: identifier, referenceType: referenceType.rawValue)
         }
+        
+        let appExtensionTarget = scheme.build.targets.compactMap { bt -> Target? in
```

Comment:
Should we instead just use `schemeTarget`?

Reply:
in my case the share extension needs both the normal app and the extension set for build targets. When either one of the targets is the app extension I set it to askForAppToLaunch otherwise the key can be nil. Using the schemeTarget doesn't have the same result.

Reply:
Ahh, okay. Let's not default it at all then. If its not specified in the `yml` we will behave like we do now.

Reply:
Sounds good to me, this way we don't enforce anything upon the user but let them take care of setting the right values in the `yml`.